frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -91,13 +91,15 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
               required
             />
           </div>
-          <div className="toolTip_dropdown">
+          <label className="control-label" htmlFor="request-size-input">
```

Comment:
`htmlFor` value is incorrect.  I don't think `OCSStorageClassDropdown` is setup correctly to use with `<label>`.

Reply:
ack